cuml/dbscan_demo.ipynb

```diff
@@ -120,19 +97,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "clustering_sk = skDBSCAN(eps=eps,\n",
-    "                         min_samples=min_samples,\n",
-    "                         algorithm=\"brute\",\n",
-    "                         n_jobs=-1)\n",
-    "clustering_sk.fit(host_data)"
+    "from sklearn.cluster import DBSCAN\n",
```

Comment:
We would usually import as skDBSCAN and cuDBSCAN to make it clear which one you're using. Typically we'd do this with imports near the top of the notebook. This would be more consistent with other notebooks and wouldn't require overwriting a name in the namespace.

Reply:
Keeping the imports in a single cell at the top makes it much easier for users to copy/paste bits of the notebook because they can grab all the imports in one go. I also tend to prefer keeping the corresponding scikit-learn and cuml imports close to each other so that the users can immediately see the similarities in the package structures.

Reply:
Hi @JohnZed ,

I understand that point, but I'd like to think in a different way.

In the future, RAPIDS users are not likely to be comparing scikit-learn against RAPIDS code, but just using RAPIDS code.

I think it is easier for them to use the proper class name (DBSCAN) rather than rename to distinguish it in this specific example.

I have performed this work in 9 of the existing notebooks, and I am more than happy to do the same work on the remaining.

In addition to that, with the new mixed approach (importing when using), there is no risk that the user makes mistakes when using sckit-learn or RAPIDS algos (because the object initialization is done in the same cell as the import).

Reply:
Hi all,

I am moving the imports to the first cell of the notebooks, and leave this discussion to a later moment. I'd love to include this notebooks to this release, and we are quite close to change freeze.

Thank you all for your comments!